isubit/luggage_people
# LUGG-828 Universal Grid Styling

See also: https://github.com/isubit/suitcase/pull/52
## Changes
- Removed grid styling from luggage_people and moved it to suitcase.css
- Removed hook_views_preprocess() that previously controlled the people grid styling
## Testing
- Add some people
- Pull down this pull request’s changes
- Make sure that you don’t have luggage_people feature locked
- Clear caches and reset all features (I also run the drush command registry-rebuild, but that might be overkill)
- Go to the people view. Is there still a nice people grid?
- Go to the expertise view. Does the area of expertise view still look the same?
- Resize the window, make sure that each people cell resizes as you change the window size. Is any text cut off or does text overflow the grid cell? Is the people position text still green?
- Go to a person node. Is it still styled?
- Again, add a person. Is the node creation form still styled?
